tests/playbooks/test_minimal.yaml

```diff
@@ -34,3 +34,4 @@
     - cinder_adoption
     - horizon_adoption
     - dataplane_adoption
+    - heat_adoption
```

Comment:
Currently in CI we use the `test_with_ceph.yaml` playbook instead of the `test_minimal.yaml`, so we should add the role there as well to test is. Logs for the run in this PR show that head adoption is not running: https://logserver.rdoproject.org/79/179/292f6f8f1d53f3ab9d8614461ddedebf48ea06b8/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/ec4747e/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2023-10-13T01:38:24EDT.log

Reply:
ack. Added now.